lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -1955,12 +1955,27 @@ void mme_app_handle_paging_timer_expiry(void* args, imsi64_t* imsi64) {
   }
   *imsi64                                = ue_context_p->emm_context._imsi64;
   ue_context_p->paging_response_timer.id = MME_APP_TIMER_INACTIVE_ID;
-  if ((mme_app_paging_request_helper(
-          ue_context_p, false, true /* s-tmsi */, CN_DOMAIN_PS)) != RETURNok) {
-    OAILOG_ERROR_UE(
-        LOG_MME_APP, ue_context_p->emm_context._imsi64,
-        "Failed to send Paging Message for ue_id " MME_UE_S1AP_ID_FMT "\n",
-        mme_ue_s1ap_id);
+  // Re-transmit Paging message only once
+  if (ue_context_p->paging_retx_count < 1) {
```

Comment:
instead of 1 can you define a parameter e.g., PAGING_RTX_LIMIT?

So we did not have a limit on retries before?